### ISSUE TITLE
/tasks/running endpoint does not return correct results

### ISSUE THREAD

**Issue by RemakingEden** [Project-MONAI/monai-deploy-workflow-manager#308]
### Description
After speaking with @jackschofield23 & @lillie-dae it seems that the /tasks/running endpoint is working incorrectly. The Mongo filter does not seem to filter tasks but seems to filter workflow instances based on task status'. Lillie and I have had it working by filtering the results after Mongo but it would be good to ensure the Mongo filter either worked or was not there to save understanding the code in the future.

### Expected behavior
The endpoint would return all tasks that are not in a "succeeded", "failed", or "cancelled" state

### Actual behavior
The endpoint returns all tasks of any instances that have tasks that equal "accepted" and dont equal "dispatched"

### Other information
I have committed tests for the /tasks endpoint as this is working correctly and will move this ticket to complete. https://github.com/Project-MONAI/monai-deploy-workflow-manager/issues/290. I have ignored the tests I have started on the /tasks/running endpoint to be finished once this change is complete. I will create a new ticket for this.
